### PR TITLE
chore(CMF): git ignore _site

### PR DESCRIPTION
**Pull request by jsomsanith**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
In `packages/cmf`a folder `_site`can be generated containing the generated cmf doc.
It is not ignored by git but it should be.

**What is the new behavior?**
Add it to gitignore

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://327.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://327.talend.surge.sh)

File: packages/cmf/.gitignore [hunk 1]
```diff
@@ -37,3 +37,4 @@ jspm_packages
 .node_repl_history
 
 /lib
+/_site
```
Comment (jmfrancois) [inline packages/cmf/.gitignore]: please add a new line
by the way your IDE should do that for you

Comment (frassinier) [inline packages/cmf/.gitignore]: done

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://327.talend.surge.sh)

**Pull request merged**